### DISCUSSION TITLE
macos install ends with efi shell prompt or reinstall prompt

### DISCUSSION THREAD

**Discussion by labaxter** [quickemu-project/quickemu#1415] [Quickemu]
Running Debian 12, having installed with the projects .deb files, I have tried to install macos ventura (and others).  The install ends with an EFI **Shell>** prompt... or it takes me back to the install screen.  

I'm following instructions here: https://github.com/quickemu-project/quickemu/wiki/03-Create-macOS-virtual-machines

I never get to this step: "On the subsequent reboots use cursor keys and enter key to select the disk you named"

I've gone through the steps to this point multiple times.

Suggestions?  What do I do at the **Shell>** prompt?

**Comment by SailAway33** [comment]
I had to go through the recovery prompt to get the drive to show. And select it as the main drive.

**Reply by labaxter** [reply]
From the "Shell>" prompt... what does one type to get there?  Or, are you suggesting some other path to "recovery?"

**Reply by makeller1** [reply]
Just ran into the same thing myself and solved with OP's recommendation. On the install screen select recovery. Then clip the apple icon in the top left. Choose your startup disk (should be the one you gave a name during the disk utility step before install). Then click restart.
